lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -194,6 +194,10 @@ class ReportRenderer {
     Util.reportJson = report;
 
     const detailsRenderer = new DetailsRenderer(this._dom);
+    const fullPageScreenshot = /** @type {LH.Artifacts.FullPageScreenshot | undefined} */ (
```

Comment:
this seems like an abuse of the `screenshot` audit details type if it's not ever used as one of those and the object is assumed to have properties its type doesn't (width and height) :/

Reply:
> this seems like an abuse of the screenshot audit details type if it's not ever used as one of those and the object is assumed to have properties its type doesn't (width and height) :/

I don't know a good answer to this is--a new audit details type? Use some non details thing instead? Make final screenshot have a width and height?--but definitely doesn't seem useful to share the type between these two places and not have them actually share the type :)

Reply:
.... extendedInfo? :P

but really; are you suggesting this should be in a debug data?

> assumed to have properties its type doesn't (width and height)

ya. and to tack on, the artifact hardcodes 0 for `timing` and `timestamp`.

Reply:
> .... extendedInfo? :P

lol

> but really; are you suggesting this should be in a debug data?

If we're going to actually use it in the report, that feels wrong too. I guess a new `audit-details` type? It feels wasteful but it's not really and it's better than repurposing something not made for this and making an existing `audit-details` type more ambiguous.

Reply:
`screenshot-with-size` ? or just `full-page-screenshot`